ampd/src/queue/queued_broadcaster.rs

```diff
@@ -38,11 +39,12 @@ impl QueuedBroadcasterDriver {
     }
 }
 
+#[automock]
 #[async_trait]
 pub trait BroadcasterClient {
     async fn broadcast<T>(&self, tx: T) -> Result
     where
-        T: Msg + Send + Sync;
+        T: Msg + Send + Sync + 'static;
```

Comment:
Having `'static` is a requirment from automock I assume?

Reply:
yep, I think it's fine to add the restriction for async